piff/psf.py

```diff
@@ -116,4 +116,28 @@ def read(cls, file_name, logger=None):
                 logger.debug("interp = %s",interp)
         return cls(model,interp)
 
+    def drawImage(self, position, offset=(0, 0), logger=None, **kwargs):
+        """Generates star given position and galsim Image
 
+        :param position:    Position we are interpolating to.
+        :param offset:      Set the nominal center of the image relative to
+                            actual center of image [Default: (0, 0)]
+        :param logger:      A logger object for logging debug info. [default: None]
+
+        :returns: a star
+        """
+        import piff
+        import galsim
+        image = galsim.Image(32, 32)
+
+        # interpolate params
+        interpolated_params = self.interp.interpolate(position)
+        # set the parameters on the psf model, which gives a new model instance
+        interpolated_model = self.model.setParameters(interpolated_params)
+        # draw on the galsim image container
+        interpolated_image = interpolated_model.drawImage(image)
```

Comment:
This should be
```
interpolated_image = interpolated_model.drawImage(image, offset=galsim.PositionD(*offset))
```
Then I would just return it.  I don't think we should be turning it into a StarData object.